tests/test_wavefront_sensing.py

```diff
@@ -32,16 +32,27 @@ def test_modulated_pyramid_wavefront_sensor():
 
 	num_steps = 12
 
-	pywfs = PyramidWavefrontSensorOptics(pupil_grid, wfs_grid, 1.0)
+	pywfs = PyramidWavefrontSensorOptics(pupil_grid, wfs_grid, 1.0, q=3)
```

Comment:
Why this change? Because of systematic noise reasons of course, but does this mean that you _need_ to have higher `q` values for the fast modulation method? If so, that should definitely be in the docstring.

Reply:
The reason for this is that the code that chooses the correct focal plane sampling of the pyramid does not work correctly. I found that out last week. That is a bug that we need to fix. I am working on that in a separate issue. In the mean time I hard-coded the correct q for this simulation.

Reply:
Okay.